tests/e2e/Services/Users/UsersBase.php

```diff
@@ -1038,29 +1038,65 @@ public function testUpdateUserPassword(array $data): array
     /**
      * @depends testGetUser
      */
-    #[Retry(count: 1)]
     public function testUpdateUserStatus(array $data): array
     {
-        /**
-         * Test for SUCCESS
-         */
-        $user = $this->client->call(Client::METHOD_PATCH, '/users/' . $data['userId'] . '/status', array_merge([
+        // Verify that the user exists
+        $response = $this->client->call(Client::METHOD_GET, '/users/' . $data['userId'], array_merge([
```

Comment:
Why do we need to test the user exists? This depends on testGetUser, so the user should already exist.

Reply:
This hasn't been addressed